### PR TITLE
Blog post: Docker CI.

### PR DESCRIPTION
**Pull request by nirgn975**
#365

### CODE REVIEW

**Comment by nirgn975** [comment]
@amitaibu I think that's pretty much ready too..

**Comment by amitaibu** [comment]
Some technical stuff:

* Convert png to jpg. Make sure width isn't higher than 800px
* Add thumb image (see previous posts to see how it looks), in the correct scale
* Add your thumb image, so it looks correct
* Add `<!-- more -->` for the teaser. The image should be part of the teaser 

design___web_development___gizra

File: app/_posts/2016-03-31-docker-ci.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Travis CI is a Free and open-source software, hosted, distributed continuous integration service used to build and test projects hosted at GitHub. Travis CI makes working in a team for a software project easier with automated builds. These builds are triggered automatically when each developer checks in (commit) their code to the repository.
+
+## CI with Docker
+In one of our projects we needed to test something we never done before - subdomains. 
+It was a pain to configure Apache on travis because every little change I wanted to make required to commit the change and push in to GitHub.
+That means that travis will build the whole project from scratch (download and install packages, download and import db, install drupal, etc) for every commit! A process that take a long time (and somethings just to get an induction if I'm on the right path).
+
+But as from August 2015 we can all use [Docker inside Travis](https://blog.travis-ci.com/2015-08-13-using-docker-on-travis-ci). Whats that mean?
```
Comment (amitaibu) [inline app/_posts/2016-03-31-docker-ci.md]: link is wrong.

**Comment by amitaibu** [comment]
Scaffolding a Drupal project from hedley, and adding your Travis config to it - would allow others (Drupal) devs to more easily follow and adapt your techniques. possible? :)

**Comment by nirgn975** [comment]
@amitaibu Ready.

**Comment by amitaibu** [comment]
I think your own avatar is still missing

**Comment by nirgn975** [comment]
@amitaibu Add my avatar (:

**Comment by amitaibu** [comment]
@nirgn975 can you please add a drupal-lamp as repo to Github as-well (`Gizra/drupal-lamp`)

**Comment by nirgn975** [comment]
@amitaibu Great!

The drupal-lamp was under Docker-lamp, but I changed the repo name to match.

**Comment by amitaibu** [comment]
@RachelBaram ready for review

**Comment by amitaibu** [comment]
Merged and deployed - thanks!

http://www.gizra.com/content/docker-travis-ci/

**Pull request closed without merging**